### DISCUSSION TITLE
No test_err in lcurve.out for dpgen-train.

### DISCUSSION THREAD

**Discussion by Edinbur** [deepmodeling/dpgen#1395] [Q&A]
Hi All,

There is always no test_err in the lcurve.out of my dpgen-train since using dpgen.
I guess it has tested every disp_freq but no results output in lcurve.out.
So, I wonder if my results are right and how to abtain the test_err in lcurve.out.
<img width="540" alt="279667188-3a45b8fb-d09c-43f8-bb34-c5145def15db" src="https://github.com/deepmodeling/dpgen/assets/92415156/b81e3200-23d4-49d9-ae9f-106f04994b85">
<img width="541" alt="279667214-628b11d0-6e47-465b-b5b4-307320090bd4" src="https://github.com/deepmodeling/dpgen/assets/92415156/aaf2ebbc-d42f-4a4c-a24a-f4bd6b8d4b66">

**Comment by njzjz** [comment]
See #649

**Comment by Edinbur** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
